src/scss/components/postLanding/about/_about.scss

```diff
@@ -1,5 +1,70 @@
 .post-landing-about{
     &__container {
         position: relative;
-  }
+        min-height: 100vh;
+    }
+
+    &__left-backdrop {
+        z-index: -1;
+        position: absolute;
+        left: 0;
+        top: 2.2em;
+    }
+
+    &__right-backdrop {
+        z-index: -1; 
+        position: absolute;
+        top: 17em;
+        right: 14em; 
+    }
+
+    &__description-container {
+        position: absolute;
+        display: flex;
+        flex-direction: column;
+        margin: calc(35em / 3) calc(50% - 4em) auto;
+        align-items: flex-start;
+
+        @include respond(tab-land) {
+            margin: calc(25em / 3) calc(50% - 1em) auto;
+        }
+
+        @include respond(tab-port) {
+            margin: calc(25em / 3) calc(50% + 1em) auto;
+        }
+    }
+
+    &__title-container {
+        margin-bottom: 4em;
+        width: 35vw;
+    }
+
+    &__title {
```

Comment:
don't make a new class for the header titles, instead @include the postlanding header mixin in the CTA & share sections!

Reply:
think up new mixins as well, a button mixin might be helpful